Dockerfile

```diff
@@ -37,11 +31,11 @@ RUN pip install .
 
 COPY docker_overlay/ /
```

Comment:
would moving this to the end of the Dockerfile help with local caching of the other layers?

Reply:
It's such an insignificant amount of time to do this operation that it doesn't matter much. Also, the overlay rarely updates